modules/core/test/v2/unit/coins/algo.ts

```diff
@@ -0,0 +1,127 @@
+import * as Promise from 'bluebird';
+const co = Promise.coroutine;
+const Wallet = require('../../../../src/v2/wallet');
+
+import 'should';
+
+const TestV2BitGo = require('../../../lib/test_bitgo');
+
+const nock = require('nock');
+nock.enableNetConnect();
+
+describe('ALGO:', function() {
+  let bitgo;
+  let basecoin;
+  let signedTxBase64;
+
+  before(function() {
+    bitgo = new TestV2BitGo({ env: 'test' });
```

Comment:
env: 'mock' in unit tests. This makes it much more obvious when you are missing a nock

Reply:
k